### ISSUE TITLE
Managing the dev server

### ISSUE THREAD

**Issue by fatimah5** [TEAMMATES/teammates#9123]
Managing the dev server ( There is no Google App Engine Standard Local Server in Run tab)

- **IntelliJ Version:** IntelliJ IDEA 2018.2.2 x64
- **Chrome Version (if applicable):** Version 69.0.3497.92

**What I was trying to do / What I need help with**
In this step:
Starting the dev server
Go to Run → Run... and select Google App Engine Standard Local Server in the pop-up box.)

 There is no Google App Engine Standard Local Server in Run tab to start!

![image](https://user-images.githubusercontent.com/41271008/45450789-dc2fac80-b69e-11e8-96d3-3c838290d9b4.png)

**Comment by xpdavid** [comment]
Click `Run...` and see if you have the option?

**Comment by fatimah5** [comment]
This what shown when I click Run, no Google App Engine Standard Local Server option appear!

![image](https://user-images.githubusercontent.com/41271008/45461463-c9ca6880-b6c8-11e8-8144-c617c6c7ba46.png)

**Comment by xpdavid** [comment]
Have you followed https://github.com/TEAMMATES/teammates/blob/master/docs/setting-up.md and https://github.com/TEAMMATES/teammates/blob/master/docs/ide-setup.md closely?

**Comment by xpdavid** [comment]
You need to wait for the index to complete before click `configure`.

You can now delete `.idea/*` folder and try import the project again.

**Comment by fatimah5** [comment]
Ok, I'll try now

**Comment by xpdavid** [comment]
Have you installed https://cloud.google.com/tools/intellij/docs/quickstart-IDEA#install plugin?

**Comment by fatimah5** [comment]
yes, I Installed and set up Cloud Tools for IntelliJ, but I didn't do the 3 steps under (before you begin) headline. Should I?

I tried the following steps from https://cloud.google.com/tools/intellij/docs/deploy-local, 
To run an App Engine application locally:
Select File > Open to open the project you want to run.
Browse to the directory containing your project.
Select Tools > Google Cloud Tools > Run on a local App Engine Standard dev server.

when I clicked run > run google app engine standard local server, there was a lot of errors as shown in the pic with this message (use -source 8 or higher to enable lambda expression Intellij Idea):

![image](https://user-images.githubusercontent.com/41271008/45467527-b679c600-b6e5-11e8-9192-1f0a0b5ceb9c.png)
